docs/application/dotnet/guides/security/privacy-related-permissions.md

```diff
@@ -17,6 +17,11 @@ The main features of the `Tizen.Security.PrivacyPrivilegeManager` class include:
 
 For a list of privacy-related privileges, see [Security and API Privileges](../../tutorials/sec-privileges.md).
 
+> [!NOTE]
+> Since Tizen 5.5, if the caller app's `App component type` is `UI app` or `Component based app` with `Frame component` then pop-ups by `RequestPermission()` and `RequestPermissions()` are launched as `group mode` with the caller app.
```

Comment:
@yl33
I have doubt here: 
1. Is it necessary to keep `App component type` is `UI app` or `Component based app` in code tags?
Kindly clarify the same.

Just in my view, can we write it like this: 
Since Tizen 5.5, if the caller app component type is UI app or component based app with frame component, then, the pop-ups by `RequestPermission()` and `RequestPermissions()` are launched as `group mode` with the caller apps.

- This is just a suggestion, can you please give a better rephrase if you don't agree to my view point.
- Also verify for the technical meaning too, in case you agree with my suggestion.

Reply:
Removed code tags.